Sample Data/ASIM/F5_ASM_ASimWebSession_IngestedLogs.csv

```diff
@@ -0,0 +1,9 @@
+"2e0bbf4a-31d9-471d-b063-6b5e5e3d7ef9","8/3/2023, 9:42:54.885 AM",F5,ASM,"11.3.0","Successful Request","Successful Request",2,,,passed,"HTTP ",,,,,,,,0,,,,,,,,,,,,,,80,"10.4.1.200",,,,"172.16.73.34","bigip-4.pme-ds.f5.com",,,,,18205860747014045699,,,,,,,,,,,"N/A",,,,,,,,,,,,,,"/",,,,GET,"Sep 19 2012 13:35:00",,,,,,,,,,,52963,"10.4.1.101",,2e769a9e1ea8b777,"N/A",,,,"device_address",,"source_address",,"destination_address","N/A","ip_address_intelligence",,,,,,,,,,200,"response_code",,,,,,,"topaz4-web4","policy_name","/Common/topaz4-web4","http_class_name","GET / HTTP/1.0\r\nUser-Agent: Wget/1.12 (linux-gnu)\r\nAccept: */*\r\nHost: 10.4.1.200\r\nConnection: Keep-Alive\r\n\r\n","full_request","N/A","attack_type","N/A","x_forwarded_for_header_value","N/A","geo_location","Sep 19 2012 11:38:36","policy_apply_date",,,,,,,,,,,,,,,,,,,,,,,,"bigip-4.pme-ds.f5.com",OpsManager,passed,,CommonSecurityLog,
+"2e0bbf4a-31d9-471d-b063-6b5e5e3d7ef9","8/2/2023, 12:26:43.427 PM",F5,ASM,"10.1.0","Illegal query string length","Illegal query string length",6,,,alerted,HTTP,,,,,,,,0,,,,,,,,,,,,,,80,"172.30.0.32",,,,"172.30.0.20","3600.lab.asm.f5net.com",,,,,416829024209663345,,,,,,,,,,,,,,,,,,,,,,,,,"/phpauction/search.php?=&q=%3Cscript%3E%3C%2Fscript%3E&=Go%21",,,,GET,"Sep 02 2009 17:09:25",,,,,,,,,,,37446,"172.30.0.126",,,,,,,,,,,,,,,,,,,,,,,200,"response_code",,,,,,,"new_app_default","policy_name","new_app","web_application_name","GET /phpauction/search.php?=&q=%3Cscript%3E%3C%2Fscript%3E&=Go%21 HTTP/1.1\r\nHost: 172.30.0.32\r\nUser-Agent: Mozilla/5.0 (X11  U  Linux i686 (x86_64)  en-US  rv:1.8.1) Gecko/20061023 SUSE/2.0-30 Firefox/2.0\r\nAccept: text/xml,application/xml,application/xhtml+xml,text/html q=0.9,text/plain q=0.8,image/png,*/* q=0.5\r\nAccept-Language: en-us,en q=0.5\r\nAccept-Encoding: gzip,deflate\r\nAccept-Charset: ISO-8859-1,utf-8 q=0.7,* q=0.7\r\nKeep-Alive: 300\r\nConnection: keep-alive\r\nReferer: http://172.30.0.32/phpauction/help.php?\r\nCookie: PHPAUCTION_SESSION=lhuqakkdn6icm9vv33p9nepcm0lga6rd  TS2ea638=1c1e60b9764bace0c13f1829c93d009ec4b6e3e4598e3bb14a9f12c67b46979e7faa5 254\r\n\r\n","full_request","Buffer Overflow","attack_type",,"x_forwarded_for_header_value","N/A","geo_location","Sep 02 2009 15:09:20","policy_apply_date",,,,,,,,,,,,,,,,,,,,,,,,"172.30.0.130",OpsManager,alerted,,CommonSecurityLog,
```

Comment:
@jayeshprajapaticrest 
Illegal query string length would be in Anomaly category?

Reply:
Can you please describe more on this?

Reply:
In one of the sample logs DeviceEventClassID = Illegal query string length. Wanted to understand the reason why it wasn't considered under the group 'AnomalyDetectionData'.

Reply:
As both types have different field configurations for CEF, We have devided events in blocks as mentioned in the document.